core/modules/system/tests/system.test

```diff
@@ -1704,66 +1704,86 @@ array_space[a b] = Value';
   }
 }
 
-
-class InfoFileTypeTest extends BackdropUnitTestCase {
+/**
+ * Ensures that all info files have expected contents.
+ */
+class InfoFileCoreTest extends BackdropUnitTestCase {
   /**
    * Tests that all core modules' .info files have the project type string set to "module".
    */
-  function testModuleInfoFileType() {
+  function testModuleInfoFileContents() {
     $modules = backdrop_system_listing('/^' . BACKDROP_PHP_FUNCTION_PATTERN . '\.module$/', 'core/modules', 'name', 0);
-    $fails = array();
+    $type_fails = array();
+    $version_fails = array();
     foreach($modules as $key => $module) {
       $info = backdrop_parse_info_file(dirname($module->uri) . '/' . $module->name . '.info');
       if (!isset($info['type'])) {
-        $fails[] = $key;
-        $this->assertTrue(FALSE, t('%key does not have project type set to "module".', array('%key' => $key)));
+        $type_fails[] = $key;
+        $this->failt(format_string('%key does not have project type set to "module".', array('%key' => $key)));
```

Comment:
Oh drat. Typo here. But since this doesn't fail if everything is okay, it didn't come up in the testing. Fixed in https://github.com/backdrop/backdrop/commit/ed38c816e4fcc8c6ece5047bb3e3d02afc92675b.